scripts/new-version-pr-generator.js

```diff
@@ -0,0 +1,180 @@
+const shopifyPolarisBotGitOverride = `GIT_COMMITTER_NAME='${polarisBotName}' GIT_COMMITTER_EMAIL='${polarisBotEmail}'`;
+
+/**
+ * BulkExecute - Run a bunch of execSync one at a time
+ *
+ * @param {array}  tasks   - List of tasks to run
+ * @param {object} options - Specific options for execSync
+ */
+function BulkExecute(tasks, options) {
+  tasks.forEach((task) => {
+    execSync(task.command, options ? options : {});
+  });
+}
+
+// Each of the repositories to create pull requests updating polaris
+const repositories = ['polaris-styleguide', 'web'];
+
+// Iterate through the repositories
+const tasks = repositories.map((repository) => {
+  // Create a new promise for each repository
+  return new Promise((resolve, reject) => {
+    try {
+      const repositoryDirectory = pathResolve(sandbox, repository);
+
+      // Message that is logged on a failed update
+      const failedUpdateMessage = `It seems your automatic branch creation for ${repository} failed. This can be due to many reasons. To resolve this follow these steps and replace [release-version] with the version you are publishing:
+
+      1. Checkout "${repository}"
+      2. Get the latest version of master "git checkout master && git pull"
+      3. Create a new branch based on the version that failed "git checkout -b update-polaris-v[release-version]"
+      4. Update "shopify/polaris" by running "npx yarn upgrade @shopify/polaris@[release-version]"
+      5. Add the changed files to git by running "git add package.json yarn.lock"
+      6. Commit the changes with "git commit -m 'Update @shopify/polaris to [release-version]'
+      7. Push the changes with "git push origin update-polaris-v[release-version]"
+      8. Create a new pull request for the branch in "${repository}"`;
+
+      // Pull request template
+      const updateBody = `
+## Version ${releaseVersion} of @shopify/polaris just got published!
+
+See what’s new: https://github.com/Shopify/polaris-react/releases/tag/${releaseVersion}
+
+cc @kaelig @dfmcphee @amrocha @BPScott
+
+---
+
+<details>
+<summary>🚨 What to do if you see “Your tests failed on CircleCI”?</summary>
+
+If tests fail, you may have to troubleshoot the problem locally.
+
+1. Checkout the \`update-polaris-${releaseVersion}\` branch:
+    \`\`\`bash
+    dev cd ${repository}
+    git fetch
+    git checkout update-polaris-${releaseVersion}
+    dev up
+    \`\`\`
+1. Apply changes/fixes
+1. Commit
+1. Push your work to the \`update-polaris-${releaseVersion}\` branch:
+    \`\`\`bash
+    git push origin update-polaris-${releaseVersion}
+    \`\`\`
+1. Repeat until the build & tests go ✅
+1. 🎩
+1. :shipit:
+</details>
+`.trim();
+
+      const updatePostObject = {
+        title: `Update @shopify/polaris to ${releaseVersion} 🚀`,
+        body: updateBody,
+        head: `update-polaris-${releaseVersion}`,
+        base: baseBranch,
+      };
+
+      // Clone the repository to the repository directory
+      execSync(
```

Comment:
Why is this not defined as a value in `commands`?

Reply:
This one needs to run in a different folder, sandbox. So it can't go in the other array of commands that run in the repository folder.